webpack/scenes/Subscriptions/SubscriptionValidations.js

```diff
@@ -12,7 +12,7 @@ export const validateQuantity = (quantity, availableQuantity) => {
   } else if (numberValue <= 0) {
     state = 'error';
     message = __('Has to be > 0');
-  } else if (availableQuantity && availableQuantity >= 0 && numberValue > availableQuantity) {
+  } else if (maxQuantity >= 0 && numberValue > maxQuantity) {
```

Comment:
Its fine to change the variable name, but you should still keep the `maxQuantity && maxQuantity >= 0 && ...` format